_includes/pbc/all/install-features/202407.0/install-the-product-management-ai-feature.md

```diff
@@ -0,0 +1,254 @@
+| ProductCategoryProductAbstractAfterUpdatePlugin | Adds action on product abstract after update event.      | | SprykerEco\Yves\ImageSearchAi\Plugin\Router |
+| ProductCategoryAbstractFormExpanderPlugin | Expands `ProductAbstractForm`.                           | | SprykerEco\Yves\ImageSearchAi\Plugin\Router |
+| ImageAltTextProductConcreteEditFormExpanderPlugin | Expands `ProductConcreteEditForm` with `alt_text` field. | | SprykerEco\Yves\ImageSearchAi\Plugin\Router |
+| ImageAltTextProductAbstractFormExpanderPlugin | Expands `ProductAbstractForm` with `alt_text` field.     | | SprykerEco\Yves\ImageSearchAi\Plugin\Router |
+| ImageAltTextProductConcreteFormExpanderPlugin | Expands `ProductConcreteForm` with `alt_text` field.     | | SprykerEco\Yves\ImageSearchAi\Plugin\Router |
+
+**src/Pyz/Zed/Product/ProductDependencyProvider.php**
+```php
+    /**
+     * The order of execution is important to support Inherited scope and sub-entity functionality
+     *
+     * @return array<\Spryker\Zed\ProductExtension\Dependency\Plugin\ProductAbstractPostCreatePluginInterface>
+     */
+    protected function getProductAbstractPostCreatePlugins(): array
+    {
+        return [
+            ...
+            new ProductCategoryProductAbstractPostCreatePlugin(),
+        ];
+    }
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return array<\Spryker\Zed\Product\Dependency\Plugin\ProductAbstractPluginUpdateInterface>
+     */
+    protected function getProductAbstractAfterUpdatePlugins(Container $container): array
+    {
+        return [
+            ...
+            new ProductCategoryProductAbstractAfterUpdatePlugin(),
+        ];
+    }
+```
+
+**src/Pyz/Zed/ProductManagement/ProductManagementDependencyProvider.php**
+
+```php
+    /**
+     * @return array<\Spryker\Zed\ProductManagementExtension\Dependency\Plugin\ProductConcreteEditFormExpanderPluginInterface>
+     */
+    protected function getProductConcreteEditFormExpanderPlugins(): array
+    {
+        return [
+            ...
+            new ImageAltTextProductConcreteEditFormExpanderPlugin(),
+        ];
+    }
+
+    /**
+     * @return array<\Spryker\Zed\ProductManagementExtension\Dependency\Plugin\ProductAbstractFormExpanderPluginInterface>
+     */
+    protected function getProductAbstractFormExpanderPlugins(): array
+    {
+        return [
+            ...
+            new ProductCategoryAbstractFormExpanderPlugin(),
+            new ImageAltTextProductAbstractFormExpanderPlugin(),
+        ];
+    }
+
+    /**
+     * @return array<\Spryker\Zed\ProductManagementExtension\Dependency\Plugin\ProductConcreteFormExpanderPluginInterface>
+     */
+    protected function getProductConcreteFormExpanderPlugins(): array
+    {
+        return [
+            ...
+            new ImageAltTextProductConcreteFormExpanderPlugin(),
+        ];
+    }
+```
+
+**src/Pyz/Zed/ProductManagement/Presentation/Add/index.twig**
```

Comment:
The same related to all the FE changes, not clear what it is, why it is needed, etc